From 55cde82204724df756a198da691471f2f3f83d5a Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Wed, 31 Mar 2021 10:08:31 +0200
Subject: [PATCH] test-install-root: create referenced targets

(cherry picked from commit cd228002ccedb927b4531a4b7dd9ea7015fdb657)

Related: #1835351
---
 src/test/test-install-root.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/test/test-install-root.c b/src/test/test-install-root.c
index fe1ca5b16f..f8b41b04db 100644
--- a/src/test/test-install-root.c
+++ b/src/test/test-install-root.c
@@ -1061,6 +1061,12 @@ int main(int argc, char *argv[]) {
         p = strjoina(root, "/usr/lib/systemd/system-preset/");
         assert_se(mkdir_p(p, 0755) >= 0);
 
+        p = strjoina(root, "/usr/lib/systemd/system/multi-user.target");
+        assert_se(write_string_file(p, "# pretty much empty", WRITE_STRING_FILE_CREATE) >= 0);
+
+        p = strjoina(root, "/usr/lib/systemd/system/graphical.target");
+        assert_se(write_string_file(p, "# pretty much empty", WRITE_STRING_FILE_CREATE) >= 0);
+
         test_basic_mask_and_enable(root);
         test_linked_units(root);
         test_default(root);
